test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -75,4 +80,19 @@ public static void assertTextFilesEqual(Path path1, Path path2) throws IOExcepti
         List<String> list2 = Files.readAllLines(path2, Charset.defaultCharset());
         assertEquals(String.join("\n", list1), String.join("\n", list2));
     }
+
+    /**
+     * Executes the command, and asserts the result message is as expected.
+     */
+    public static void assertCommandResult(Command command, AddressBook addressBook,
+                                           AddressBook expectedAddressBook, String expectedMessage) {
```

Comment:
but in this case why there is false negative? previously I passed in the `addressbook` after executing the command. Then how to check the false negatives